packages/lisk-transactions/src/utils/sign_and_verify.ts

```diff
@@ -110,7 +106,7 @@ export const verifyMultisignatures = (
 				return;
 			}
 
-			const { verified: signatureVerified } = verifySignature(
+			const { valid: signatureVerified } = validateSignature(
```

Comment:
It's weird that the function call is `validateSignature` but the function responds with an object property named `signatureVerified`